sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeedRollupType.cs

```diff
@@ -5,8 +5,28 @@
 
 namespace Azure.AI.MetricsAdvisor.Models
 {
+    /// <summary>
+    /// The strategy used by a <see cref="DataFeed"/> when rolling-up the ingested data before
+    /// detecting anomalies.
+    /// </summary>
     [CodeGenModel("NeedRollupEnum")]
     public readonly partial struct DataFeedRollupType
     {
+        /// <summary>
+        /// No roll-up will be applied for anomaly detection.
```

Comment:
what does rolling-up means?

Reply:
It's a kind of data pre-processing, such as summing/counting values, getting max/min values in a set, etc.
I gave Chris an example above for another question:

| City | Product | Cost |
| --- | --- | --- |
| Redmond | Shoes | 1.00 |
| Redmond | Hat | 2.00 |
| Seattle | Shoes | 3.00 |
| Seattle | Hat | 4.00 |
| AVG | Shoes | 2.00 |
| AVG | Hat | 3.00 |
| Redmond | AVG | 1.50 |
| Seattle | AVG | 3.50 |
| AVG | AVG | 2.50 |

The entries with `AVG` represent rolled-up data:
| City | Product | Cost |
| --- | --- | --- |
| AVG | Shoes | 2.00 |
means that the average cost of shoes across all cities in my data source (Redmond and Seattle) is 2.00.

It's a well-known term, so I think there's no need for extra explanation on the SDK side.

Reply:
Like summarize/aggregate? 
Doing quick search summarize seems like a more common concept than roll-up. Might be worth exploring a name change

Reply:
> Like summarize/aggregate?

Yep. I'm not sure if the terms are exactly the same, though, since I don't have much experience with it.

It's still pretty used, though. If you search for `rollup data`, you'll see the name is used across multiple databases (including databases that Metrics Advisor support).